### DISCUSSION TITLE
BUG： Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by XinzeWu** [community/community#52990] [GitHub Education]
GitHub saied "Your uploaded proof has already been used **previously." but what should I use to pass it ?

**Comment by samus-aran** [comment]
Hi @XinzeWu 

Just checking your public profile I can see you are using non English language and perhaps you need to pre-translate your evidence in order for the application process to progress.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by shanirinacohen** [reply]
Hi i got the same response for GitHub.
My husband (dviroscohen) has a Github account, so i thought to use his account, so I added my academic mail to his account and used my Student Id, but got a message that it's not him. So I removed my mail from there and then i've created a new account with my mail and my academic mail, but now when i'm trying to send the student id it says that it was already used for another account. 
But my husband removed it from his account! So it's like no one is using it. I need fast support here.

**Reply by juniorxxue** [reply]
Same response. I keep receiving the "location bug" https://github.com/orgs/community/discussions/42246" until I ran into this one after several tries with different browsers/machines.

**Comment by anzenchitai** [comment]
It just told me to go complete my profile after submitting my school ID then I signed out and logged in again as it exactly told me then applied again.

Now I am getting this response:
"Your uploaded proof has already been used previously"

Sorry, what? You guys seriously need to get this "student pack" !#@? together.

**Comment by MelGod** [comment]
Re-take a photo of your ID and make sure you have the two-step authentication.
